src/main/java/cz/muni/fi/entities/Car.java

```diff
@@ -0,0 +1,83 @@
+package cz.muni.fi.entities;
+
+/**
+ * Class represents car
+ * @author Lucie Kureckova, 445264
+ */
+
+@Entity
+public class Car {
+    
+    @Id
+    @GeneratedValue(strategy=GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    private Person driver;
+    @OneToOne
+    private Component engine;
+    @OneToOne
+    private Component aerodynamics;
+    @OneToOne
+    private Component suspension;
+    @OneToOne
+    private Component transmission;
+    @OneToOne
+    private Component brakes;
+
+    public Car(Person driver, Component engine, Component aerodynamics, Component transmission, Component brakes) {
```

Comment:
preferoval by som asi prezdny konstruktor a vsetko nastavoval cez settre

Reply:
preco ? 
je podla mna lepsie ked takto vidis co vsetko musis nastavit aby si mal auto

Reply:
takze nechat byt tak jak je ci smazat ci pridat konstruktor s zadnymi argumenty?

Reply:
pretoze tam nieje tam validacia bud ju tam prida pre kazdy condition alebo to odstrani, resp. si tam moze volat pre kazdy component jeho setter